docs/src/SelectList.doc.js

```diff
@@ -20,185 +41,361 @@ card(
+/>`}
+      />
+    </MainSection.Subsection>
+    <MainSection.Subsection>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        description="Keep the same type of selection for a group of items. An example of this might be a filter bar. If some items could use SelectList and some items need to use Dropdown, use Dropdown for all the items in the group."
+        defaultCode={`
+<Flex>
+  <SelectList
+    id="selectlistexample"
+    onChange={() => {}}
+    options={[
+      {label:'Filter value 1', value: 'fv11'},
+      {label:'Filter value 2', value: 'fv12'},
+    ]}
+    placeholder='Filter 1'
+  />
+  <SelectList
+    id="selectlistexample"
+    onChange={() => {}}
+    options={[
+      {label:'Filter value 1', value: 'fv21'},
+      {label:'Filter value 2', value: 'fv22'},
+    ]}
+    placeholder='Filter 2'
+  />
+</Flex>`}
+      />
+      <MainSection.Card
+        cardSize="md"
+        type="don't"
+        description="Mix Dropdown and SelectList in a group of items."
+        defaultCode={`
+<Flex>
+  <SelectList
+    id="selectlistexample"
+    onChange={() => {}}
+    options={[
+      {label:'Filter value 1', value: 'fv11'},
+      {label:'Filter value 2', value: 'fv12'},
+    ]}
+    placeholder='SelectList'
+  />
+  <Button
+    accessibilityControls="basic-dropdown-example"
+    accessibilityHaspopup
+    iconEnd="arrow-down"
+    text="Dropdown"
+    inline
+  />
+</Flex>`}
+      />
+    </MainSection.Subsection>
+  </MainSection>,
+);
+
+card(
+  <MainSection name="Accessibility">
+    <MainSection.Subsection
+      title="Labels"
+      description={`
+      SelectList comes with Label built-in: just use the \`label\` prop. Be sure to provide a unique id so the Label is associated with the correct SelectList.`}
     />
-  );
-}
-    `}
-  />,
+  </MainSection>,
 );
 
 card(
-  <Example
-    id="helperText"
-    name="Example: Helper Text"
-    description={`Whenever you want to provide more information about a form field, you should use \`helperText\`.`}
-    defaultCode={`
-function Example(props) {
-  const [country, setCountry] = React.useState('belgium')
-  const countries = [
-    {
-      value: "belgium",
-      label: "Belgium"
-    },
-    {
-      value: "france",
-      label: "France"
-    },
-    {
-      value: "usa",
-      label: "USA"
-    }
-  ];
-  return (
-    <Box padding={2} color="white">
-      <SelectList
-        id="country"
-        name="country"
-        onChange={({ value }) => setCountry(value)}
-        options={countries}
-        placeholder="Select country"
-        label="Country"
-        helperText="Pick your favourite country (the one with the best fries)"
-        value={country}
+  <MainSection name="Variants">
+    <MainSection.Subsection title="Size">
+      <MainSection.Card
+        cardSize="md"
+        title="Large"
+        description={`Use \`lg\` as the default size within Pinterest products.`}
+        defaultCode={`<SelectList
```

Comment:
If you look at the code in the example sandbox, the formatting is a bit funky. I think you need to start with a line break (so `<SelectList` is on the next line with no spaces preceding it), then space the remaining lines accordingly. (It looks like this is the case with all the examples that show a code sandbox)

Reply:
Finally got to this one. Thanks for the patience.